calyx-opt/src/pass_manager.rs

```diff
@@ -162,7 +165,7 @@ impl PassManager {
         });
 
         // Push all aliases
-        let mut aliases = self.aliases.iter().collect::<Vec<_>>();
+        let mut aliases = self.aliases_cmd.iter().collect::<Vec<_>>();
```

Comment:
What does this change help with?

Reply:
It allows printing the alias pass name instead of the fully expanded pass. So the help message at "all" will be "validate,pre-opt..." instead of the long chain of passes.